app/src/main/java/com/example/dine_and_donate/ProfileActivity.java

```diff
@@ -147,4 +129,16 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
             }
         });
     }
+
+    private void navigationHelper(Class activity) {
```

Comment:
it seems like this isn't used anywhere? you can delete it if so

Reply:
used in the setUpBottomNav() method